tests/cypress/support/commands.js

```diff
@@ -146,38 +146,38 @@ Cypress.Commands.add('openTaskJob', (taskName, jobID = 0, removeAnnotations = tr
     cy.openJob(jobID, removeAnnotations);
 });
 
+Cypress.Commands.add('interactControlButton', (objectType) => {
+    cy.get('body').click('top');
+    cy.get(`.cvat-${objectType}-control`).trigger('mouseleave').trigger('mouseout').trigger('mouseover');
```

Comment:
.click() is not enough?

Reply:
cy.click () contains a certain number of mouse events. Perhaps this is one of the reasons for incorrect interaction with the button. I think trigger() as a lower-level command will be more optimal.

Reply:
> cy.click () contains a certain number of mouse events. 

What do you mean? cy.click just invokes click function of the element. You can do the same using  window.document.body.click() as far as I understand.

Reply:
cy.click() contains events:
```
pointerover
mouseover
pointermove
mousemove
pointerdown
mousedown
pointerup
mouseup
click
```